Architect-Coders/MU8
# Issue #20: Integrate firebase crashlytics

**Implementation of crashlytics:** #20 
- Crashlyticis functionality added

**Testing Done:**

* On Emulator: Huawei P20 Lite.

**How To Test:**

In the activity add these lines code

````
   val crashButton = Button(this)
   crashButton.text = "Crash!"
   crashButton.setOnClickListener {
      Crashlytics.getInstance().crash() // Force a crash
   }

    addContentView(crashButton, ViewGroup.LayoutParams(
    ViewGroup.LayoutParams.MATCH_PARENT,
    ViewGroup.LayoutParams.WRAP_CONTENT))
````

compile the application and press the button that appears on the screen, When you press the button the application will close. to look at the fault that has been generated you have to enter [Firebase](https://console.firebase.google.com/u/0/) and look for the MU8APP project, when you find the project look for the Crashlytics module and the error report will appear there.

![Captura de Pantalla 2019-12-03 a la(s) 12 26 07 a  m](https://user-images.githubusercontent.com/57641701/70022829-87a67480-1563-11ea-8937-cace69912b7a.png)

**Signed-off-by:**

`Santiago Lozada <santiagolozada95@gmail.com>`
